### ISSUE TITLE
Is the shuffle parameter of train_loader set to False purposely?

### ISSUE THREAD

**Issue by yuxinyuan** [NVIDIA/waveglow#145]
https://github.com/NVIDIA/waveglow/blob/6188a1d106a1060336040db82f464d6441f39e21/train.py#L97

I notice that the shuffle parameter is set to False for DataLoader. Is there any reason behind it ?

**Comment by rafaelvalle** [comment]
DistributedSampler has it's own shuffling scheme. Closing due to inactivity.
